### ISSUE TITLE
Provide a way not to auto enable and start systemd units

### ISSUE THREAD

**Issue by nemanja-boric-sociomantic** [jordansissel/fpm#1363]
In case template unit files are added to `-deb-systemd`, fpm will try to enable and start this service, which is not possible. There should be an attribute to prevent auto-enable & start of the service, and leave it to the user. Perhaps just the `systemctl reload` should be mandatory.

**Comment by cam-fulcrum** [comment]
Yeah, unfortunately this overrides the `exit 101` trick too https://jpetazzo.github.io/2013/10/06/policy-rc-d-do-not-start-services-automatically/

**Comment by jordansissel** [comment]
Agreed.

To add this setting, you'd want to look in the [templates directory](https://github.com/jordansissel/fpm/blob/master/templates/deb/) for things like [this](https://github.com/jordansissel/fpm/blob/master/templates/deb/postinst_upgrade.sh.erb#L35-L37) (where systemd start/enable are invoked)
